packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx

```diff
@@ -419,7 +419,7 @@ export const Autocomplete = observer((properties: AutocompleteProps): JSX.Elemen
 														onClick={(e: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => onTermClick && onTermClick(e)}
 														href={term.url.href}
 														{...valueProps}
```

Comment:
Seems to be a slight conflict in the valueProps here and in all other focusable elements within autocomplete.

Previously the onMouseOver event would eventually trigger a focus event:
https://github.com/searchspring/snap/blob/v0.39.2/packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx#L204-L214

Now however, it will happen instantaneously, and the valueProps are perhaps unnecessary?